requirements.in

```diff
@@ -34,6 +34,7 @@ python-slugify < 1.2.0
 raven
 statsd
 unicodecsv
+venusian
```

Comment:
This will have to go into `setup.py` because it's a memex dependency (`requirements.in` is only for h). It's a bit confusing while we have these two project in the same repo, sorry.